### PR TITLE
volume not exposed if not set

### PR DESCRIPTION
**Pull request by hmert**

### CODE REVIEW

File: src/Ojs/SiteBundle/Resources/views/Issue/detail.html.twig [hunk 1]
```diff
@@ -29,14 +29,18 @@
             %}
                 <h1>{{ fieldDisplay(issue, 'title') }}</h1>
                 <h4>
-                    {% if issue.volume is not null and issue.volume != 0 and issue.volume != "0"%}
+                    {{ dump(issue.volume) }}
```
Comment (behram) [inline src/Ojs/SiteBundle/Resources/views/Issue/detail.html.twig]: @hmert can you remove dump function in this line. 😄

Comment (hmert) [inline src/Ojs/SiteBundle/Resources/views/Issue/detail.html.twig]: DUMP
image

Comment (behram) [inline src/Ojs/SiteBundle/Resources/views/Issue/detail.html.twig]: thx.

**Comment by behram** [comment]
i am merging now.

**Pull request merged**